### ISSUE TITLE
Can't manage to make it run.

### ISSUE THREAD

**Issue by Adraesh** [itsTeckel/vu#5]
Hello!

Bellow the logs:
```
Skipping update as '/vu/client/vu.com' exists.
(EE)
Fatal server error:
(EE) Server is already active for display 1
	If this server is no longer running, remove /tmp/.X1-lock
	and start again.
(EE)
Found server.key
Found Battlefield 3 client files
0050:err:explorer:initialize_display_settings Failed to query current display settings for L"\\\\.\\DISPLAY1".
Starting Z:\vu\client\vu.com
wine: Unhandled page fault on read access to 000000B8 at address 00C4653A (thread 0104), starting debugger...
010c:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
010c:err:winediag:nodrv_CreateWindow Make sure that your X server is running and that $DISPLAY is set correctly.
Unhandled exception: page fault on read access to 0x000000b8 in 32-bit code (0x00c4653a).
Register dump:
 CS:0023 SS:002b DS:002b ES:002b FS:0063 GS:006b
 EIP:00c4653a ESP:022afbc4 EBP:022afbc8 EFLAGS:00010202(  R- --  I   - - - )
 EAX:000000b8 EBX:01116240 ECX:000000b8 EDX:ffffffff
 ESI:00000000 EDI:00000000
Stack dump:
0x022afbc4:  000000b8 022afbdc 00c463a1 022afcac
0x022afbd4:  01ec78d8 01da7db8 022afbec 00c44693
0x022afbe4:  000000b8 022afcac 022afc00 00c73b31
0x022afbf4:  000000b8 00000000 00000000 022afd1c
0x022afc04:  00e23a4d 022afcac 3256007b 022afc24
0x022afc14:  00cc59e7 01da86e0 01101008 022afcc4
Backtrace:
=>0 0x00c4653a EntryPoint+0xffffffff() in eacore (0x022afbc8)
  1 0x00c463a1 EntryPoint+0xffffffff() in eacore (0x022afbdc)
  2 0x00c44693 EntryPoint+0xffffffff() in eacore (0x022afbec)
  3 0x00c73b31 EntryPoint+0xffffffff() in eacore (0x022afc00)
  4 0x00e23a4d EntryPoint+0x4f63a() in eacore (0x022afd1c)
  5 0x00e24e9e EntryPoint+0x50a8b() in eacore (0x022afd74)
  6 0x00e1b03d EntryPoint+0x46c2a() in eacore (0x022afe30)
  7 0x00e1b704 EntryPoint+0x472f1() in eacore (0x022afed8)
  8 0x00e1bc52 EntryPoint+0x4783f() in eacore (0x022afee4)
  9 0x00e123eb EntryPoint+0x3dfd8() in eacore (0x022aff28)
  10 0x00e124a5 EntryPoint+0x3e092() in eacore (0x022aff30)
  11 0x7b6560a0 in kernel32 (+0x3609f) (0x022aff48)
  12 0x7bc56ae7 EntryPoint+0xffffffff() in ntdll (0x022aff5c)
  13 0x7bc56c98 EntryPoint+0xffffffff() in ntdll (0x022affec)
0x00c4653a EntryPoint+0xffffffff in eacore: movl	0x0(%eax),%eax
Modules:
Module	Address			Debug info	Name (59 modules)
PE	  400000-  420000	Deferred        eacoreserver
PE	  420000-  7e4000	Deferred        ole32
PE	  c40000- 17a8000	Export          eacore
PE	 17b0000- 19c3000	Deferred        comdlg32
PE	 19d0000- 1d92000	Deferred        comctl32
PE	10000000-100d8000	Deferred        ucrtbase
PE	61b00000-61b85000	Deferred        sechost
PE	61f80000-61f83000	Deferred        api-ms-win-core-debug-l1-1-0
PE	62040000-62043000	Deferred        api-ms-win-core-sysinfo-l1-1-0
PE	62140000-6215b000	Deferred        wtsapi32
PE	62340000-623bd000	Deferred        uxtheme
PE	62800000-628b8000	Deferred        usp10
PE	62d00000-62dbc000	Deferred        advapi32
PE	64540000-64626000	Deferred        winhttp
PE	64740000-64743000	Deferred        api-ms-win-core-file-l2-1-0
PE	64b40000-64b78000	Deferred        shcore
PE	653c0000-653c3000	Deferred        api-ms-win-core-errorhandling-l1-1-0
PE	65440000-65443000	Deferred        api-ms-win-core-processthreads-l1-1-0
PE	65980000-6599a000	Deferred        version
PE	66980000-66983000	Deferred        api-ms-win-core-string-l1-1-0
PE	67800000-6791b000	Deferred        combase
PE	68380000-68383000	Deferred        api-ms-win-core-processenvironment-l1-1-0
PE	68540000-68543000	Deferred        api-ms-win-core-util-l1-1-0
PE	68ac0000-68ac3000	Deferred        api-ms-win-core-rtlsupport-l1-1-0
PE	68c40000-68d3a000	Deferred        shlwapi
PE	69980000-69983000	Deferred        api-ms-win-core-memory-l1-1-0
PE	69cc0000-69cc3000	Deferred        api-ms-win-core-profile-l1-1-0
PE	6a200000-6a203000	Deferred        api-ms-win-core-datetime-l1-1-0
PE	6a300000-6a64e000	Deferred        oleaut32
PE	6a6c0000-6a6c3000	Deferred        api-ms-win-core-handle-l1-1-0
PE	6a900000-6aa7e000	Deferred        setupapi
PE	6aec0000-6aecf000	Deferred        psapi
PE	6af80000-6af83000	Deferred        api-ms-win-core-namedpipe-l1-1-0
PE	6b400000-6b403000	Deferred        api-ms-win-core-libraryloader-l1-1-0
PE	6b4c0000-6b659000	Deferred        wininet
PE	6ba80000-6ba8c000	Deferred        api-ms-win-core-fibers-l1-1-1
PE	6bec0000-6bec4000	Deferred        api-ms-win-core-file-l1-1-0
PE	6c0c0000-6c0fa000	Deferred        imm32
PE	6c500000-6c503000	Deferred        api-ms-win-core-heap-l1-1-0
PE	6c780000-6c783000	Deferred        api-ms-win-core-synch-l1-1-0
PE	6dbc0000-6dc0a000	Deferred        mpr
PE	6e540000-6e543000	Deferred        api-ms-win-core-synch-l1-2-0
PE	6e8c0000-6e8cd000	Deferred        api-ms-win-core-localization-l1-2-1
PE	6e9c0000-6e9c3000	Deferred        api-ms-win-core-file-l1-2-0
PE	6f840000-6f843000	Deferred        api-ms-win-core-processthreads-l1-1-1
PE	6fa80000-6fa83000	Deferred        api-ms-win-core-localization-l1-2-0
PE	6fdc0000-6ff91000	Deferred        rpcrt4
PE	70900000-70903000	Deferred        api-ms-win-core-interlocked-l1-1-0
PE	70a00000-70a03000	Deferred        api-ms-win-core-timezone-l1-1-0
PE	70b80000-70b83000	Deferred        api-ms-win-core-console-l1-1-0
PE	71440000-71483000	Deferred        jsproxy
PE	7b000000-7b2b3000	Deferred        kernelbase
PE	7b620000-7b7dc000	Export          kernel32
PE	7bc00000-7be5a000	Dwarf           ntdll
PE	7dec0000-7decb000	Deferred        winspool
PE	7df30000-7e807000	Deferred        shell32
PE	7e920000-7e924000	Deferred        ws2_32
PE	7eb10000-7eb17000	Deferred        gdi32
PE	7ec70000-7ed58000	Deferred        user32
Threads:
process  tid      prio (all id:s are in hex)
00000038 services.exe
	0000003c    0
	00000040    0
	00000054    0
	00000058    0
	00000064    0
	00000078    0
	0000007c    0
	00000094    0
	000000c0    0
00000044 winedevice.exe
	00000048    0
	0000005c    0
	00000060    0
0000004c explorer.exe
	00000050    0
	00000070    0
00000068 plugplay.exe
	0000006c    0
	00000080    0
	00000084    0
	00000088    0
	000000a4    0
	000000a8    0
0000008c winedevice.exe
	00000090    0
	00000098    0
	0000009c    0
	000000a0    0
	000000ac    0
	000000b0    0
000000b8 rpcss.exe
	000000bc    0
	000000c4    0
	000000c8    0
	000000cc    0
	000000d0    0
	000000d4    0
	000000d8    0
000000ec vu.com
	000000f0    0
000000f8 (D) Z:\vu\bf3\Core\EACoreServer.exe
	000000fc    0
	00000104    0 <==
	00000110    0
System information:
    Wine build: wine-5.17
    Platform: i386
    Version: Windows 7
    Host system: Linux
    Host version: 5.4.0-53-generic
Start waiting on wineserver
waiting ...
waiting ...
waiting ...
waiting ...
waiting ...
wineserver completed
```

Thank yoi.

**Comment by benjmd** [comment]
Mine looks the same as yours, or just loops forever on `waiting ...`
